includes/Modules/Analytics/Web_Tag.php

```diff
@@ -187,6 +187,11 @@ protected function enqueue_gtag_script() {
 					),
 					$tag
 				);
+
+				$snippet_comment_begin = sprintf( "\n<!-- %s -->\n", esc_html__( 'Google Analytics snippet added by Site Kit', 'google-site-kit' ) );
+				$snippet_comment_end   = sprintf( "\n<!-- %s -->\n", esc_html__( 'End Google Analytics snippet added by Site Kit', 'google-site-kit' ) );
+
+				return $snippet_comment_begin . $tag . $snippet_comment_end;
```

Comment:
Per my last comment regarding wrapping the tag html separately – this is what I meant ☝️ 😄 

About this instance specifically, the search and replace that happens above is only done when enabled by cookie consent type plugins require that Analytics isn't loaded until consent is granted so it won't always run.

Let's update the function here to always be added so that we can add our before/after comments, but it should still only do the search/replace based on `$block_on_consent_attrs`. This wraps the entire snippet, including the configuration very nicely 
![image](https://user-images.githubusercontent.com/1621608/131326911-7113e9c5-93e2-4aa0-a269-56a829b68735.png)